test/support/api/states.js

```diff
@@ -15,12 +15,17 @@ export function getState() {
 
 const states = getStates();
 
-// Exporting one of each state.status
-export const stateTodo = _.find(states, { id: '22222' });
-export const stateInProgress = _.find(states, { id: '33333' });
-export const stateDone = _.find(states, { id: '55555' });
-export const stateUnableToComplete = _.find(states, { id: '66666' });
-export const stateThmgTransfered = _.find(states, { id: '77777' });
+function getStateByName(name) {
+  return _.find(states, state => {
+    return state.attributes.name === name;
```

Comment:
The equivalent of `role.name` is actually `role.slug`. Annoying discrepancy, but there it is.

Reply:
Realized my typo, corrected. State slugs are enforced as unique, just like role name, and can be used as an identifier.